### DISCUSSION TITLE
如何在没有GUI的linux服务器上挂载pikpak?

### DISCUSSION THREAD

**Discussion by TyroneVeblen** [AlistGo/alist#7497] [Q&A] [answered]
我按照https://github.com/AlistGo/alist/pull/7024#issue-2470412018进行挂载。但是一直提示Click here。
即使我填入提示Click here时所给出的a标签中的device_id，以及在windows端过验证码后的Captcha token。依旧提示需要验证。。。
现在完全没头绪了，求大佬帮忙

**Comment by pongfcnkl** [comment, marked as answer]
不用账号密码，只填刷新令牌和设备id

**Reply by TyroneVeblen** [reply]
谢谢，起作用了。非常感谢
